README.md

```diff
@@ -101,34 +101,21 @@ spec:
       uri: "https://github.com/openshift-psap/kvc-simple-kmod.git"
     type: Git
     dockerfile: |
-      FROM DRIVER_TOOLKIT_IMAGE
+      FROM <driver-toolkit image> as builder
 
       WORKDIR /build/
 
-      # In case additional packages are required, you need to manually add the yum repos to the container
+      RUN git clone https://github.com/openshift-psap/simple-kmod.git
 
-      # Expecting kmod software version as an input to the build
-      ARG KMODVER
+      WORKDIR /build/simple-kmod
 
-      # Grab the software from upstream
-      RUN git clone https://github.com/openshift-psap/simple-kmod.git
-      WORKDIR simple-kmod
-
-      # Prep and build the module
-      RUN make all       KVER=$(rpm -q --qf "%{VERSION}-%{RELEASE}.%{ARCH}"  kernel-core) KMODVER=${KMODVER} \
-      && make install   KVER=$(rpm -q --qf "%{VERSION}-%{RELEASE}.%{ARCH}"  kernel-core) KMODVER=${KMODVER}
-
-      # Add the helper tools
-      WORKDIR /root/kvc-simple-kmod
-      ADD Makefile .
-      ADD simple-kmod-lib.sh .
-      ADD simple-kmod-wrapper.sh .
-      ADD simple-kmod.conf .
-      RUN mkdir -p /usr/lib/kvc/ \
-      && mkdir -p /etc/kvc/ \
-      && make install
-
-      RUN systemctl enable kmods-via-containers@simple-kmod
+      RUN make all install # [KVER=<kenel version if not default>] [KMODVER=<kernel module version if not default>]
+
+      FROM registry.access.redhat.com/ubi8:latest
+
+      RUN dnf -y install kmod
+
+      COPY --from=builder /lib/modules/<kernel version>/* /lib/modules/<kernel version>/
```

Comment:
They also need to copy etc/driver-toolkit-release.json from the builder

Reply:
> They also need to copy etc/driver-toolkit-release.json from the builder

What is it needed for @yevgeny-shnaidman?

Reply:
What is the difference between putting it in `/opt/lib/modules` rather than in `/lib/modules`?
I think `modprobe` is expecting it to be in `/lib/modules`. Anyway if we make that change let's keep it outside of this PR - this one is only supposed to split the image.

Reply:
> > They also need to copy etc/driver-toolkit-release.json from the builder
> 
> What is it needed for @yevgeny-shnaidman? What does it contains that isn't present in `/etc/os-release`?

it does not contain full kernel version, for example, that is needed for preflight.

Reply:
> v2 mounts /lib/modules into the DriverContainer. That helps if the OOT module depends on an in-tree one

If a driver is in-tree, won't it be available on the driver-container without mounting anything?

> You can override that with -b

That means we need to update the DS doc as well

> Fair enough

https://issues.redhat.com/browse/MGMT-10194

Reply:
> it does not contain full kernel version, for example, that is needed for preflight

Then I will add it.

Reply:
Added `driver-toolkit-release.json`